packages/page/src/UserInterface/Command/InitializeHomepageCommand.php

```diff
@@ -0,0 +1,125 @@
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\HandleTrait;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+/**
+ * @internal your code should not create direct dependencies on this implementation
+ *           projects can utilize the `sulu:page:initialize` command to
+ *           initialize new webspaces with the homepage
+ *
+ * @final
+ */
+#[AsCommand(name: 'sulu:page:initialize', description: 'Initializes the homepage per webspace locale')]
+class InitializeHomepageCommand extends Command
+{
+    use HandleTrait;
+
+    public function __construct(
+        private WebspaceManagerInterface $webspaceManager,
+        private PageRepositoryInterface $pageRepository,
+        MessageBusInterface $messageBus
+    ) {
+        parent::__construct();
+        $this->messageBus = $messageBus;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $ui = new SymfonyStyle($input, $output);
+
+        $webspaces = $this->webspaceManager->getWebspaceCollection();
+        foreach ($webspaces as $webspace) {
+            $localizations = $webspace->getLocalizations();
+            foreach ($localizations as $localization) {
+                if ($this->homepageExists($webspace, $localization->getLocale())) {
+                    $ui->info(
+                        \sprintf(
+                            'Homepage for locale "%s" in webspace "%s" already exists',
+                            $localization->getLocale(),
+                            $webspace->getKey()
+                        )
+                    );
+                    continue;
+                }
+                $homepage = $this->createHomepage($webspace, $localization->getLocale());
+                $this->publishHomepage($homepage->getUuid(), $localization->getLocale());
+            }
+        }
+        $ui->success('Homepage initialization completed');
+
+        return Command::SUCCESS;
+    }
+
+    private function homepageExists(Webspace $webspace, string $locale): bool
+    {
+        $result = $this->pageRepository->countBy([
+            'webspaceKey' => $webspace->getKey(),
+            'locale' => $locale,
+            'parentId' => null,
+            'stage' => DimensionContentInterface::STAGE_LIVE,
+        ]);
+
+        return $result > 0;
+    }
+
+    private function createHomepage(Webspace $webspace, string $locale): PageInterface
+    {
+        $webspaceKey = $webspace->getKey();
+        $message = new CreatePageMessage(
+            $webspaceKey,
+            CreatePageMessageHandler::HOMEPAGE_PARENT_ID,
+            [
+                'title' => $webspace->getName(),
```

Comment:
awesome we discussed today this should be webspace name not the homepage like the old builder. You already did it 💪